performance_tests/asteroid_test.amlg

```diff
@@ -1,13 +1,13 @@
 (seq
 	#howso (null)
-	(direct_assign_to_entities (assoc howso (load "../howso.amlg")))
-	(assign_to_entities (assoc filepath "../"))
+	(load_entity "../howso.amlg" "howso" (false) (false))
+	(call_entity "howso" "initialize" (assoc trainee_id "model"))
+	(direct_assign_to_entities (assoc howso (retrieve_entity_root "howso")))
+	(destroy_entities "howso")
```

Comment:
nah, no reason to add a method to return this code -  I'll just update the tests to use the same format as unit tests, so it'll be a 'howso' entity instead of the trainee code being directly in the test file